jerry-core/ecma/base/ecma-helpers.c

```diff
@@ -472,12 +471,10 @@ ecma_find_internal_property (ecma_object_t *object_p, /**< object descriptor */
        property_p != NULL;
        property_p = ECMA_GET_POINTER (ecma_property_t, property_p->next_property_p))
   {
-    if (property_p->type == ECMA_PROPERTY_INTERNAL)
+    if (property_p->flags & ECMA_PROPERTY_FLAG_INTERNAL
```

Comment:
I'd suggest using some more parentheses to make operator precedence explicit. (Just like in other parts of the patch.)